<!--REQUEST_ID=072661ae-0be9-412a-9ee6-07a3d3c5fb8f-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.51259&attempt_id=feb45ec0-b29c-48ec-ae81-c77dca1f2379)

Attempted: reptyr

<details><summary>Partial log (click to expand)</summary><p>

```
build flags: SHELL=/nix/store/bpspdmsl0yys24gs70flsvcw3wcnl7lx-bash-4.4-p23/bin/bash PREFIX=\$\(out\)
gcc -Wall -Werror -D_GNU_SOURCE -g   -c -o reptyr.o reptyr.c
In file included from platform/platform.h:28:0,
                 from reptyr.c:37:
platform/../ptrace.h:66:14: error: field 'user' has incomplete type
  struct user user;
              ^~~~
make: *** [<builtin>: reptyr.o] Error 1
builder for '/nix/store/4lx8qfb8sik1y32dspanm90sv9ahrhcj-reptyr-0.6.2.drv' failed with exit code 2
error: build of '/nix/store/4lx8qfb8sik1y32dspanm90sv9ahrhcj-reptyr-0.6.2.drv' failed
```
</p></details>

--- a/pkgs/os-specific/linux/reptyr/default.nix
+++ b/pkgs/os-specific/linux/reptyr/default.nix
@@ -29,5 +29,6 @@ stdenv.mkDerivation rec {
     maintainers = with lib.maintainers; [raskin];
     license = lib.licenses.mit;
     description = "Reparent a running program to a new terminal";
+    homepage = https://github.com/nelhage/reptyr;
   };
 }

--- a/pkgs/os-specific/linux/reptyr/default.nix
+++ b/pkgs/os-specific/linux/reptyr/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, lib, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
   version = "0.6.2";
@@ -20,9 +20,14 @@ stdenv.mkDerivation rec {
 
   makeFlags = ["PREFIX=$(out)"];
   meta = {
-    platforms = [ "i686-linux" "x86_64-linux" ];
-    maintainers = with stdenv.lib.maintainers; [raskin];
-    license = stdenv.lib.licenses.mit;
-    description = ''A Linux tool to change controlling pty of a process'';
+    platforms = [
+      "i686-linux"
+      "x86_64-linux"
+      "i686-freebsd"
+      "x86_64-freebsd"
+    ] ++ lib.platforms.arm;
+    maintainers = with lib.maintainers; [raskin];
+    license = lib.licenses.mit;
+    description = "Reparent a running program to a new terminal";
   };
 }

